### PR TITLE
logout attempt 1 - failing

### PR DESCRIPTION
**Pull request by mariel-delagarza**
# UPDATE
can confirm 
- user can create self by entering username
- user can click on "logout" and get kicked back to login page
- a manual page refresh confirms user is logged out

Screen Recording 2021-03-05 at 7 30 33 PM

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -5,9 +5,12 @@
     <div>
       <%= link_to "Create game", new_game_path %>
     </div>
+    
+    <div>
+    <%= link_to "Log out", logout_path, method: "delete" %>
```
Comment (horrorsaur) [inline app/views/home/index.html.erb:10]: I could be wrong, but I think the method has to be passed a symbol.

... , method: :delete

Does your rails routes output show the path for that action?

**Comment by mariel-delagarza** [comment]
just kidding it's only fixed if it's inside actions -- i went to put it back next to "Create Game" annnnd it's back to not finding "logout" inside the ApplicationController 🙃

File: config/routes.rb [hunk 2]
```diff
@@ -6,8 +6,9 @@
   get "/welcome", to: "users#new", as: :login
   resources :users, only: :create
 
-  resources :games, only: [:new, :create]
+  resources :games, only: [:new, :create, :delete]
   get "/:token", to: "games#show", as: :game
   patch "/:token", to: "games#update"
   get "/:token/settings", to: "games#edit", as: :edit_game
+  get '/:token/logout', to: "games#logout", as: :logout
```
Comment (efatsi) [inline config/routes.rb:13]: i would recommend making this route point to the sessions controller. something like: 
```rb
get "/logout", to: "sessions#destroy", as: :logout
```
Associating user logouts with game functionality is a bit disjointed

File: config/routes.rb [hunk 3]
```diff
@@ -6,8 +6,9 @@
   get "/welcome", to: "users#new", as: :login
   resources :users, only: :create
 
-  resources :games, only: [:new, :create]
+  resources :games, only: [:new, :create, :delete]
```
Comment (efatsi) [inline config/routes.rb:9]: revert this pretty please

File: config/routes.rb (hunk 2 above)
Comment (mariel-delagarza) [reply config/routes.rb:13]: okay that makes sense i didn't see a sessions controller and wasn't sure if i should make that happen gracias

**Review by horrorsaur — APPROVED** [review]
Looks great! Depending on how this pans out, I think the sessions_controller could be extended for additional functionality.

**Pull request merged**